spec/support/verifiable.rb

```diff
@@ -179,4 +179,66 @@
     end
   end
 
+  describe "#overrided_methods" do
```

Comment:
This is not consistent with rspec standards or the codebase.. Also "override" is not the verb agreed upon, but `skip`.